lib/utils.pm

```diff
@@ -1221,4 +1223,25 @@ sub script_run_interactive {
     }
 }
 
+sub setup_autofs_server {
+    my ($autofs_conf_file, $autofs_map_file, $test_conf_file, $test_conf_file_content, $test_mount_dir) = @_;
```

Comment:
```suggestion
    my (%args) = @_;
```
when having such huge list of variable it is really bad idea on rely on position only , much convenient way is to use hash and pass values via sane variables names

Reply:
done ;)!